soql-analyzer/src/main/scala/com/socrata/soql/AnalysisDeserializer.scala

```diff
@@ -301,7 +313,11 @@ class AnalysisDeserializer[C, T](columnDeserializer: String => C, typeDeserializ
         val dictionary = DeserializationDictionaryImpl.fromInput(cis)
         val deserializer = new Deserializer(cis, dictionary, v)
         deserializer.read()
-      case v@CurrentVersion =>
+      case v@CurrentVersion  =>
```

Comment:
It looks like the handling of these three case matches (this one and above and below) are identical? why are they broken out?

Reply:
I could have combined that into one case.  It just isn't very straight forward.  The new case block will be removed soon after too.